roles/README.adoc

```diff
@@ -163,6 +207,16 @@ If some variables need to be parameterized according to distribution and version
   when: __rolename_vars_file is file
 ----
 
+* Add this as the first task in `tasks/main.yml`:
+
```

Comment:
There should be a `+` on this line so the code block is lined up with the bullet.